eddy/core/project.py

```diff
@@ -183,6 +184,22 @@ def session(self) -> 'Session':
         """
         return cast('Session', self.parent())
 
+    @property
+    def name(self):
+        try:
+            parse(str(self.ontologyIRI), rule='IRI')
+            name = str(self.ontologyIRI)
+        except ValueError:
+            name = "http://obda.org/"+self.oldName
+        simpleName = self.getIRI(name).getSimpleName()
+        return simpleName if len(simpleName) > 0 else self.oldName
```

Comment:
Sto notando che se si una un IRI dell'ontologia che termina con `#` o `/` questo metodo fallisce sempre (viene restituito l'oldName) e non si riesce a far aggiornare il nome in quanto `getSimpleName()` restituisce la stringa vuota su queste IRI. Sto cercando di capire se è un problema di `getSimpleName` o se possiamo fare qualcosa per evitarlo.